library/default-synths-extra.scd

```diff
@@ -114,11 +111,11 @@
 // again using "n" for some variation on frequency, "decay" for scaling noise duration relative to tonal part
 // "sustain" for overall timescale, "accelerate" for tonal glide
 (
-	SynthDef(\supersnare, {|out, sustain=1, pan, accelerate, n, decay=1 |
-		var env, sound, accel;
+	SynthDef(\supersnare, {|out, sustain=1, pan, accelerate, n, decay=1, speed=1|
+		var env, sound, scale;
 		env = EnvGen.ar(Env.linen(0.01, 0, 0.6, 1, -3), timeScale:sustain, doneAction:2);
-		accel = Line.kr(1, 1+accelerate, 0.2);
-		sound = LPF.ar(Pulse.ar(100*accel*(n/5+1).wrap(0.5,2)), Line.ar(1030, 30, 0.2*sustain));
+		scale = DirtFreqScale.kr(speed, accelerate, sustain);
```

Comment:
I think it would be better to have one variable called `freq`:

```
freq = 100 * DirtFreqScale.kr(speed, accelerate, sustain)
[…] Pulse.ar(freq * (n / 5 + 1).wrap(0.5, 2))